### ISSUE TITLE
Spiral cut

### ISSUE THREAD

**Issue by MARIOBASZ** [vlachoudis/bCNC#983]
I have written some lines of code for a complement, which allows in an alternative the spiral cut. The last pass is circular:
![image](https://user-images.githubusercontent.com/11743403/46209387-1d14fb80-c304-11e8-8807-c78cca9b21e4.png)

What I write is called a splice generator, the variables to enter are:
center 1, center 2, radius 1, radius 2, angle 1, angle 2 (angle is separated by 2: phi and alpha 1 and 2 respectively, using the trochoidal concepts of @harvie plugin). It allows other alternatives that only Arquimedes spiral

**Comment by sfinexer** [comment]
It is ingenious, it is so possible to make deepening a thin mill!

**Comment by MARIOBASZ** [comment]
Use as soft splice of curves
![image](https://user-images.githubusercontent.com/11743403/46212230-db884e80-c30b-11e8-9263-c158c9ecf461.png)

curiosity or use as ... peacock tail ??

![image](https://user-images.githubusercontent.com/11743403/46212298-083c6600-c30c-11e8-9add-e9c5edfa1881.png)

**Comment by MARIOBASZ** [comment]
@sfinexer , Yes, I find it useful in my work (I'm going slow) version of trochoidal cut: I descend in helix and then enlarging the hole.
It could also be another alternative for pocket operation (especially at high speeds to smooth out jolts in passages from one diameter to the next as it may happen now) or a small contribution for adaptive cuts.
